### PR TITLE
DG update: Email Command

### PR DESCRIPTION
**Pull request by samleewy**
Added implementation for Email Command in Developer Guide.

Things to take note:
1. Added validation for 0 students in both diagrams, however, not implemented in code yet, as mentioned in #95.
2. There's merge conflicts, will resolve merge conflicts once PR has been approved.

PR #101 🎉

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -133,6 +133,30 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
...
+5. Upon `EmailCommand#execute()`, a list of `Student` are retrieved through `Model#getFilteredStudentList()`
+6. The list of `Student` emails are then concatenated with ';' into a `String`
+7. The concatenated `String` is then returned to `LogicManager` as a new `CommandResult`
+8. The `CommandResult` containing the concatenated email is then displayed to the user through `ResultDisplay`
+
+The following activity diagram summarizes what happens when a user executes the `emails` command:
+![EmailCommandActivityDiagram.png](images/sam/EmailCommandActivityDiagram.png)
+
+The following sequence diagram summarizes what happens when a user executes the `emails` command:
+![EmailCommandSequenceDiagram.png](images/sam/EmailCommandSequenceDiagram.png)
```
Comment (enhao25) [inline docs/DeveloperGuide.md:181]: Hmmm I think line 154 to 155 and line 157 to 158 are duplicates ah.

**Review by enhao25 — CHANGES_REQUESTED** [review]
Yeap LGTM, just repeated lines under the DG which could be removed eh. Else looks good to be approved.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (samleewy) [reply docs/DeveloperGuide.md:181]: It's activity and sequence diagram! hahaha

Comment (enhao25) [reply docs/DeveloperGuide.md:181]: O.o damnnn u did both. Haha ok sorry. Let me go approve ah

**Review by enhao25 — APPROVED** [review]
LGTM

**Review by JonahhGohh — APPROVED** [review]
LGTM! 
image

What do you having this condition as "valid command input format" instead? I feel like "command exist" is not very clear 😅

**Comment by samleewy** [comment]
> LGTM!
> image
> 
> What do you having this condition as "valid command input format" instead? I feel like "command exist" is not very clear 😅

@JonahhGohh ok will change, thanks!

**Pull request merged**